src/middlewares/cross-origin/index.js

```diff
@@ -0,0 +1,8 @@
+const crossOrigin = (req, res, next) => {
+	res.header('Access-Control-Allow-Origin', '*');
+	res.header('Access-Control-Allow-Methods', 'GET, PUT, POST, PATCH, DELETE, SEARCH');
+	res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept, Authorization');
+	next();
+};
+
+module.exports = crossOrigin;
```

Comment:
if code is shared between multiple controllers :
- put into middleware (controller).  - routes + middleware + controller
- Or put into model layer